packages/react-scripts/scripts/init.js

```diff
@@ -277,6 +277,17 @@ module.exports = {
    */
   noncePlaceholder: '',
 
+  /**
+   * Configuration for GraphQL codegen. Used by \`client-preset\` babel plugin to deduplicate
+   * query documents in built output.
+   */
+  graphQL: {
+    // Local directory of GraphQL codegen output.
+    artifactDirectory: '',
+    // Function name for tagged GraphQL documents in code.
+    gqlTagName: 'graphql',
+  },
+
```

Comment:
This just updates the `.dagster.js` template. Our existing apps should just be updated manually.